docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma/cancer_model.rst

```diff
@@ -279,24 +279,28 @@ data_dir = J:/Project/simulation_science/multiple_myeloma/data/cause_model_input
    * - incidence_MM_first_relapse
      - MM
      - MM_first_relapse
-     - data_dir/incidence First-line.csv
+     - data_dir/incidence First-line.csv - data_dir/mortality First-line.csv
```

Comment:
@mattkappel @alibow Since we have updated incidence and mortality csv files, do you want me to do the subtraction and output new files for relapse rate?